src/NServiceBus.Core/Serializers/SimpleJson/EnumAwareStrategy.cs

```diff
@@ -7,6 +7,8 @@
     // https://github.com/facebook-csharp-sdk/simple-json/issues/15
     class EnumAwareStrategy : PocoJsonSerializerStrategy
     {
+        EnumAwareStrategy() {}
```

Comment:
what is this for?

Reply:
Do you mean the ctor? To make sure it only can be created in the singleton initializer

Reply:
I see. Wasn't sure we would have to explicitly restrict the creation but no strong opinion against that.

Reply:
> Inheritors of PocoSerializationStrategy need to be singletons because of the reflection caches.

See PR description ;)